packages/vector_graphics/lib/src/listener.dart

```diff
@@ -95,19 +117,26 @@ class FlutterVectorGraphicsListener extends VectorGraphicsCodecListener {
   final TextDirection? _textDirection;
 
   final ui.PictureRecorder _recorder;
-  final ui.Canvas _canvas;
+  ui.Canvas _canvas;
```

Comment:
Can this be final again? I think you're mostly using the _patternCanvases right?

Reply:
yes, done.